server/tests/helpers/utils.js

```diff
@@ -49,3 +35,47 @@ export const getUserId = async (email, username) => {
     console.log(error);
   }
 };
+
+export const loginUser = agent => new Promise((resolve) => {
+  agent
```

Comment:
Good job Soji, I think we would not need this now that we can seed data 😄

Reply:
Okay thanks, I need this to check for a user that is logged in but not yet verified such that the user will not be able to create, delete or update an article if he is yet to verify his account.

Reply:
Base on the logic, no user can log in without being verified. I think we might not need this function. or what do you think?

Reply:
Thank you Tosin, A user does not have to be verified to necessarily be able to log in, the user will just be limited in the actions he can perform such that he will only be able to view articles and not be able to create, update, delete or even comment until verified, so we still need it.